app-text/libepubgen/libepubgen-0.0.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="EPUB generator for librevenge"
+DESCRIPTION="libepubgen is an EPUB generator for librevenge. It supports \
+librevenge's text document interface and--currently in a very limited \
+way--presentation and vector drawing interfaces."
+HOMEPAGE="https://sourceforge.net/projects/libepubgen"
+COPYRIGHT="2016 David Tardon"
+LICENSE="MPL v2.0"
+REVISION="1"
+SOURCE_URI="https://heanet.dl.sourceforge.net/project/libepubgen/libepubgen-$portVersion/libepubgen-$portVersion.tar.gz"
+CHECKSUM_SHA256="966f903626a5cea06e197e93d2b11c36c8ccc67655392dc8445856168bb9fe68"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libepubgen$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libepubgen_0.0$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	librevenge$secondaryArchSuffix
+	boost$secondaryArchSuffix
+	cppunit$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libepubgen${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libepubgen_0.0$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	libepubgen$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	librevenge${secondaryArchSuffix}_devel
+	boost${secondaryArchSuffix}_devel
+	cppunit${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
```

Comment:
**`haiku${secondaryArchSuffix}_devel`** is not needed in BUILD_**PRE**REQUIRES. Having it in BUILD_REQUIRES is enough ;-)